sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ExprUtils.scala

```diff
@@ -140,4 +144,80 @@ object ExprUtils extends QueryErrorsBase {
       TypeCheckSuccess
     }
   }
+
+  def assertValidAggregation(a: Aggregate): Unit = {
+    def checkValidAggregateExpression(expr: Expression): Unit = expr match {
+      case expr: AggregateExpression =>
+        val aggFunction = expr.aggregateFunction
+        aggFunction.children.foreach {
+          child =>
```

Comment:
code style nit to save one indentation level. It's also the style of the previous code: https://github.com/apache/spark/pull/41763/files#diff-583171e935b2dc349378063a5841c5b98b30a2d57ac3743a9eccfe7bffcb8f2aL432
```
aggFunction.children.foreach { child =>
  ...
}
```